go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -247,7 +247,7 @@ func runOvnKube(ctx *cli.Context) error {
 		// since we capture some metrics in Start()
 		metrics.RegisterMasterMetrics(ovnNBClient, ovnSBClient)
 
-		ovnController := ovn.NewOvnController(clientset, factory, stopChan, nil, ovnNBClient, ovnSBClient)
+		ovnController := ovn.NewOvnController(clientset, egressIPClientset, factory, stopChan, nil, ovnNBClient, ovnSBClient, util.EventRecorder(clientset))
```

Comment:
@alexanderConstantinescu why not just make the EventRecorder internally in NewOvnControler using the clientset it already has?